app/code/Magento/Inventory/etc/db_schema_whitelist.json

```diff
@@ -1,87 +1,91 @@
 {
-  "inventory_source": {
-    "column": {
-      "source_code": true,
-      "name": true,
-      "enabled": true,
-      "description": true,
-      "latitude": true,
-      "longitude": true,
-      "country_id": true,
-      "region_id": true,
-      "region": true,
-      "city": true,
-      "street": true,
-      "postcode": true,
-      "contact_name": true,
-      "email": true,
-      "phone": true,
-      "fax": true,
-      "use_default_carrier_config": true
+    "inventory_source": {
+        "column": {
+            "source_code": true,
+            "name": true,
+            "enabled": true,
+            "description": true,
+            "latitude": true,
+            "longitude": true,
+            "country_id": true,
+            "region_id": true,
+            "region": true,
+            "city": true,
+            "street": true,
+            "postcode": true,
+            "contact_name": true,
+            "email": true,
+            "phone": true,
+            "fax": true,
+            "use_default_carrier_config": true
+        },
+        "constraint": {
+            "PRIMARY": true
+        }
     },
-    "constraint": {
-      "PRIMARY": true
-    }
-  },
-  "inventory_source_carrier_link": {
-    "column": {
-      "link_id": true,
-      "source_code": true,
-      "carrier_code": true,
-      "position": true
+    "inventory_source_carrier_link": {
+        "column": {
+            "link_id": true,
+            "source_code": true,
+            "carrier_code": true,
+            "position": true
+        },
+        "constraint": {
+            "PRIMARY": true,
+            "INV_SOURCE_CARRIER_LNK_SOURCE_CODE_INV_SOURCE_SOURCE_CODE": true
+        }
     },
-    "constraint": {
-      "PRIMARY": true,
-      "INV_SOURCE_CARRIER_LNK_SOURCE_CODE_INV_SOURCE_SOURCE_CODE": true
-    }
-  },
-  "inventory_source_item": {
-    "column": {
-      "source_item_id": true,
-      "source_code": true,
-      "sku": true,
-      "quantity": true,
-      "status": true
+    "inventory_source_item": {
+        "column": {
+            "source_item_id": true,
+            "source_code": true,
+            "sku": true,
+            "quantity": true,
+            "status": true
+        },
+        "constraint": {
+            "PRIMARY": true,
+            "INVENTORY_SOURCE_ITEM_SOURCE_CODE_INVENTORY_SOURCE_SOURCE_CODE": true,
+            "INVENTORY_SOURCE_ITEM_SOURCE_CODE_SKU": true
+        },
+        "index": {
+            "INVENTORY_SOURCE_ITEM_SOURCE_CODE_SKU_QUANTITY": true,
```

Comment:
we don't have this index declared `INVENTORY_SOURCE_ITEM_SOURCE_CODE_SKU_QUANTITY `
@bartekszymanski